classes/user-key-store.bbclass

```diff
@@ -426,7 +426,7 @@ def set_keys_dir(name, d):
     if d.getVar(name + '_KEYS_DIR', True) == d.getVar('SAMPLE_' + name + '_KEYS_DIR', True):
         d.setVar(name + '_KEYS_DIR', d.getVar('DEPLOY_DIR_IMAGE', True) + '/user-keys/' + name.lower() + '_keys')
 
-python () {
+python check_deploy_keys() {
```

Comment:
I call it with the followings from proper packagegroup/bb files:

+ inherit user-key-store
+ do_install[postfuncs] += "check_deploy_keys "